.gitlab-ci.yml

```diff
@@ -10,11 +10,11 @@ variables:
     description: |-
       Version of mender-artifact for the builder image - 'master' or a specific version
   MENDER_CLIENT_VERSION:
-    value: "latest"
+    value: "master"
```

Comment:
Commit message, did you actually mean: "The new packages won't land in `stable` APT distributions **until** after the release"?